tests/cpp/operator/mkldnn.cc

```diff
@@ -77,4 +77,11 @@ TEST(MKLDNN_UTIL_FUNC, AlignMem) {
   LOG(INFO) << "Skipped for GCC " << __GNUC__ << "." << __GNUC_MINOR__;
 #endif
 }
+
+TEST(MKLDNN_UTIL_FUNC, MemFormat) {
+  // Check whether the number of format is correct.
+  CHECK_EQ(mkldnn_format_last, 56);
```

Comment:
Is mkldnn_format_last an enum or constant? If so, you can use static_assert<> somewhere in the code  and it doesn't have to be a unit test.

Reply:
Yes, it's an enum. This request is from @marcoabreu. Please refer https://github.com/apache/incubator-mxnet/pull/9918#discussion_r171472242 and [jira#98](https://issues.apache.org/jira/browse/MXNET-98) for the background of this. Thanks.